cli/internal/codegen/testdata/expected_typescript.ts

```diff
@@ -104,18 +38,22 @@ export namespace svc {
         }
 
         public DummyAPI(params: Request): Promise<void> {
-            return this.baseClient.doVoid("POST", ` + "`/svc.DummyAPI`" + `, params)
+            return this.baseClient.doVoid("POST", `/svc.DummyAPI`, params)
         }
 
         public Get(params: GetRequest): Promise<void> {
             const query: any[] = [
                 "boo", params.Baz,
             ]
-            return this.baseClient.doVoid("GET", ` + "`/svc.Get?${encodeQuery(query)}`" + `)
+            return this.baseClient.doVoid("GET", `/svc.Get?${encodeQuery(query)}`)
         }
 
         public RESTPath(a: string, b: number): Promise<void> {
-            return this.baseClient.doVoid("GET", ` + "`/path/${a}/${b}`" + `)
+            return this.baseClient.doVoid("GET", `/path/${a}/${b}`)
+        }
+
+        public TupleInputOutput(params: Tuple<string, WrappedRequest>): Promise<Tuple<boolean, Foo>> {
+            return this.baseClient.do<Tuple<boolean, Foo>>("POST", `/svc.TupleInputOutput`, params)
```

Comment:
I'm not sure we actually need to specify the type in the call to `do()` here; I think it's inferred from the return type of the surrounding function. But might not hurt either.

Reply:
I think on generated code it's ok to be more explicit - just incase there's an edge case with inference in the future that we can avoid now